src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -100,6 +100,66 @@
                                               (swap! state assoc :deleting? true)
                                               (react/call :rm-mc this))}])})
 
+(defn- clear-overlay [state refs]
+       (swap! state assoc :publishing? false))
```

Comment:
Doesn't use `refs`.  Also, you can use dissoc, since `nil` evaluates to false:
`(swap! state dissoc :publishing?)`